Core/Frameworks/Baikal/WWWRoot/index.php

```diff
@@ -49,16 +49,18 @@
 \Flake\Framework::bootstrap();
 
 # Bootstrapping Baïkal
-\Baikal\Framework::bootstrap();
+//\Baikal\Framework::bootstrap();
```

Comment:
Why did you keep this comment?

Reply:
oh, that was when i was doing tests, deleted it on e8320ff738823ba87fb559a2c323e53a1727a34e